components/match2/wikis/rainbowsix/match_summary.lua

```diff
@@ -308,6 +311,34 @@ function MapVeto:create()
 	return self.root
 end
 
+-- Custom Caster Class
+local Casters = Class.new(
+	function(self)
+		self.root = mw.html.create('div')
+			:addClass('brkts-popup-comment')
+			:css('white-space','normal')
+			:css('font-size','85%')
+		self.casters = {}
+	end
+)
+
+function Casters:addCaster(caster)
+	if Logic.isNotEmpty(caster) then
+		local nameDisplay = '[[' .. caster.name .. '|' .. caster.displayName .. ']]'
+		if caster.flag then
+			table.insert(self.casters, Flags.Icon(caster['flag']) .. ' ' .. nameDisplay)
+		else
+			table.insert(self.casters, nameDisplay)
+		end
+	end
+	return self
+end
+
+function Casters:create()
+	return self.root
+		:wikitext('Caster' .. (#self.casters > 1 and 's' or '') .. ': ')
+		:wikitext(table.concat(self.casters, #self.casters > 2 and ', ' or ' & '))
```

Comment:
just as an fyi this doesn't make it use the `&` between second to last and last opponent in case of >3 opponents

Reply:
lets say you have
```lua
self.casters = {'a', 'b', 'c'}
```
then `table.concat(self.casters, #self.casters > 2 and ', ' or ' & ')` will produce: `'a, b, c'`
while `mw.text.listToText(self.casters, ', ', ' & ')` will produce: `'a, b & c'`